ocs_ci/utility/lvmo_utils.py

```diff
@@ -56,3 +63,68 @@ def lvmo_health_check(tries=20, delay=30):
         delay=delay,
         backoff=1,
     )(lvmo_health_check_base)()
+
+
+def get_sno_disks_by_path():
+    """
+    Get list of storage devices by it's path as listed on node
+
+    Returns:
+        list: list of storage devices full-path (str)
+    """
+    storage_disks_count = config.DEPLOYMENT.get("lvmo_disks")
+    oc_obj = OCP()
+    disks = oc_obj.exec_oc_debug_cmd(
+        node=constants.SNO_NODE_NAME, cmd_list=[LS_DEVICE_BY_PATH]
+    )
+    raw_disks_list = disks.split("\n")
+    raw_disks_list = list(filter(None, raw_disks_list))
+    disks_by_path = list()
+    for line in raw_disks_list[-storage_disks_count:]:
+        if "sda" not in line:
+            disk_name = "/dev/disk/by-path/" + line
+            disks_by_path.append(disk_name)
+
+    return disks_by_path
+
+
+def get_sno_blockdevices():
+    """
+    Gets list of storage devices by it's names
+
+    Returns:
+        list: list of storage devices full-names (str)
+
+    """
+    storage_disks_count = config.DEPLOYMENT.get("lvmo_disks")
+    disks_by_name = list()
+    oc_obj = OCP()
+    disks = oc_obj.exec_oc_debug_cmd(
+        node=constants.SNO_NODE_NAME, cmd_list=["lsblk --json"]
+    )
+    disks = json.loads(disks)
+    for disk in range(1, (storage_disks_count + 1)):
+        disk_name = "/dev/" + (disks["blockdevices"][disk]["name"])
+        disks_by_name.append(disk_name)
+
+    return disks_by_name
+
+
+def delete_lvm_cluster():
+    """
+    Delete lvm cluster if exists
+
+    raise:
+        execption if lvmcluster cant be deleted
+    """
+    clean_all_test_projects()
+    lmvcluster = OCP(kind="LVMCluster", namespace=constants.OPENSHIFT_STORAGE_NAMESPACE)
+    try:
+        lmvcluster.delete(resource_name=constants.LVMCLUSTER)
+    except CommandFailed as e:
+        if f'lvmclusters.lvm.topolvm.io "{constants.LVMCLUSTER}" not found' not in str(
+            e
+        ):
+            raise e
+        else:
+            log.info("LVMCluster not found, procced with creation of new one")
```

Comment:
Why this log line is in else block it should be in the if itself.

Reply:
i want test to continue running in case lvm cluster is not exist, or raise exception in other cases that it fails to delete it